styles/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    font-family: 'Roboto', sans-serif;
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+h1 {
+    display: flex;
+    justify-content: center;
+    width: 306px;
+    height: 47px;
+    margin: 120px 567px 0 567px;
+}
+
+hr {
+    border: none;
+    background-color: #F5BD41;
+    height: 4px;
+    width: 100px;
+    margin: 20px 670px;
+}
+
+.main {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-top: 60px;
+    margin-bottom: 100px;
+}
+
+.box {
+    width: 296px;
+    height: 176px;
+}
+
+.headphones img {
+    width: 75px;
+    height: 92px;
+    padding: 5px 106px 50px;
+}
+
+.headphones p {
+    width: 141px;
+    height: 28px;
+    font-size: 24px;
+    padding: 0 77px 0 78px;
+}
+
+.chat img {
+    width: 84px;
+    height: 84px;
+    padding: 5px 106px 50px;
+}
+
+.chat p {
+    width: 296px;
+    height: 28px;
+    font-size: 24px;
+    padding: 6px 0 0;
+    text-align: center;
+
+}
+
+.laptop img {
+    width: 100px;
+    height: 67px;
+    padding: 5px 106px 50px;
+}
+
+.laptop p {
+    width: 168px;
+    height: 28px;
+    font-size: 24px;
+    text-align: center;
+    padding-top: 25px;
+
+}
+
+.ok img {
+    width: 92px;
+    height: 83px;
+    padding: 5px 106px 50px;
+}
+
+.ok p {
+    width: 225px;
+    height: 28px;
+    font-size: 24px;
+    text-align: center;
+    padding-top: 8px;
+}
+
+.Blue {
+    color: white;
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1440px;
+    padding: 120px 0 200px 104px;
+    background-color: #10367c;
```

Comment:
according to design your background must  cover all scrren, so for that you can create new div with class and transfer backgrounfd color there